vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -254,6 +254,31 @@
     </ClCompile>
     <ClCompile Include="WebSocketResourceFactory.cpp" />
   </ItemGroup>
+  <ItemGroup Condition="'$(UseFabric)' == 'true'">
+    <ClCompile Include="..\Microsoft.ReactNative\Modules\Animated\AdditionAnimatedNode.cpp" />
```

Comment:
If these sources are to be compiled into both Desktop and Universal dlls, shouldn't they be placed in, say, `Shared.vcxitems`?

Reply:
They have different conditions.  universal uses these sources even when fabric isn't enabled.  Desktop only uses them for fabric.